server/controllers/usersController.js

```diff
@@ -6,73 +6,66 @@ class UsersController {
 	static async createUser(req, res) {
 		try {
 			const {
-				firstName,
-				lastName,
+				firstname,
+				lastname,
 				email,
 				password,
 				nationality,
 				telephone,
-				workAddress,
-				isAdmin
+				workaddress,
+				isadmin
 			} = req.body;
 			const userPassword = Auth.hashPassword(password);
 			const regUser = await new UserModel({
-				firstName,
-				lastName,
+				firstname,
+				lastname,
 				email,
 				password: userPassword,
 				nationality,
 				telephone,
-				workAddress,
+				workaddress,
 				status: 'unverified',
-				isAdmin
+				isadmin
 			});
 			if (!regUser.createUser()) {
 				res
 					.status(statusCode.CONFLICT)
 					.send({ status: statusCode.CONFLICT, error: 'Email already in use' });
 			}
-			const token = Auth.generateToken(regUser.isadmin, regUser.email);
+			const token = Auth.generateToken(regUser.isadmin, regUser.email, regUser.firstname, regUser.lastname);
```

Comment:
The token should have `isAdmin` and `email` or just `email` only

Reply:
I use the isAdmin to verify if the user is admin and I planned to use the email in loan endpoints to identify the user.

Reply:
Did you understand my comments above?

Reply:
I get it now. Working on it right away